### ISSUE TITLE
Move to Open Telemetry

### ISSUE THREAD

**Issue by mythi** [intel/cri-resource-manager#238]
I've started to draft the move in my branch mythi/cri-resource-manager/tree/otel but the work cannot be completed until Open Telemetry defines the Views API we currently depend on (via OpenCensus): https://github.com/open-telemetry/oteps/pull/89

**Comment by klihub** [comment]
Let's wait then for opentelemetry to mature/gain all the functionality we need before switching over.

**Comment by mythi** [comment]
Yes, agree. I created this ticket so that we keep an eye on the progress every now and then.
